python/lsst/sims/photUtils/LSSTdefaults.py

```diff
@@ -9,7 +9,8 @@ class LSSTdefaults(object):
     """
 
     def __init__(self):
-        self._seeing = {'u': 0.77, 'g':0.73, 'r':0.70, 'i':0.67, 'z':0.65, 'y':0.63}
+        #self._seeing = {'u': 0.77, 'g':0.73, 'r':0.70, 'i':0.67, 'z':0.65, 'y':0.63}
```

Comment:
You should update the docstring on the seeing attribute of LSSTdefaults to reflect the new meaning of "seeing" (also the fact that these values are no longer consistent with arXiv:0805.2366)